sdk/ml/azure-ai-ml/azure/ai/ml/operations/_registry_operations.py

```diff
@@ -134,3 +134,18 @@ def begin_create(
         )
 
         return poller
+
+
+    # @monitor_with_activity(logger, "Registry.Delete", ActivityType.PUBLICAPI)
+    def delete(self, *, name: str, **kwargs: Dict) -> None:
+        """Delete a registry. Returns nothing on a successful operation.
+
+        :param name: Name of the registry
+        :type name: str
+        """
+        resource_group = kwargs.get("resource_group") or self._resource_group_name
```

Comment:
Nit: make this an explicit arg and document it.

Reply:
Dropping this - the resource group is extracted from the operations object itself. This pattern is applied to all our other operation classes, and I don't see why we should break that pattern here.